### ISSUE TITLE
command Monitor and crontab dont work

### ISSUE THREAD

**Issue by kestouf** [GameServerManagers/LinuxGSM#2799]
## User Story

As a [user description], I want [desired action] so that [desired outcome].

## Basic info

* **Distro:** [Ubuntu 18.04]
* **Game:** [Minecraft]
* **Command:** [Monitor]
* **LinuxGSM version:** [v12.34.56]

## Further Information
Hello
I'm looking to do a crontab to launch the monitor on a Minecraft server.
On a Linux machine, I have 3 users: Server1, Server2, Server3
Each user runs a different Minecraft server with different ports.
I made for each user a different crontab
*/3 * * * *  /home/ Server1/minecraft/mcserver monitor > /dev/null 2>&1
*/3 * * * *  /home/ Server2/minecraft/mcserver monitor > /dev/null 2>&1
*/3 * * * *  /home/ Server3/minecraft/mcserver monitor > /dev/null 2>&1

I don't know why but none of them work. So I did a test by adding crontab
*/3 * * * *  /home/ Server1/minecraft/mcserver monitor > /home/ Server1/minecraft/log.txt

and in the log.txt files:

^[[K[ .... ] Monitor mcserver: Checking lockfile: ^[[36mCHECKING^[[0m
^[[K[^[[31mERROR ^[[0m] Monitor mcserver: Checking lockfile: No lockfile found:$
* Start mcserver to run monitor.

Can you help me please ?

Thx
Best regard

**Comment by dgibbs64** [comment]
`./mcserver start`

**Comment by kestouf** [comment]
Hello
I'm sorry, but I don't understand your answer.
The command you write in your response is just to start the server.

**Comment by dgibbs64** [comment]
correct you only get the error if you don't start the server first.
